server/resp/src/main/java/org/infinispan/server/resp/commands/list/LLEN.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.server.resp.commands.list;
+
+import io.netty.channel.ChannelHandlerContext;
+import org.infinispan.multimap.impl.EmbeddedMultimapListCache;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespErrorUtil;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+import org.infinispan.util.concurrent.CompletionStages;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * @link https://redis.io/commands/llen/
+ *
+ * Returns the length of the list stored at key. If key does not exist,
+ * it is interpreted as an empty list and 0 is returned.
+ * When the value at key is not a list, an error is returned.
+ *
+ * @since 15.0
+ */
+public class LLEN extends RespCommand implements Resp3Command {
+   public LLEN() {
+      super(2, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
+                                                      ChannelHandlerContext ctx,
+                                                      List<byte[]> arguments) {
+
+      if (arguments.size() < 1) {
```

Comment:
Should we throw an error if argument size is > 1 as well?

Reply:
Coming to think of it, we should add something to validate the arguments with the command arity.

Reply:
Yeah, that should work

Reply:
yes, I agree. I will do a PR with that in mind so we don't need to code this anymore

Reply:
https://issues.redhat.com/browse/ISPN-14896